### DISCUSSION TITLE
is there any logic for providers that can't be reached at the time of sync?

### DISCUSSION THREAD

**Discussion by aniel300** [euzu/tuliprox#114] [General]
is there any logic for providers that can't be reached at the time of sync?

**Comment by euzu** [comment]
If you configured telegram than you receive a message.
But there is no try later option.
The next scheduled time is when the update starts again.

**Reply by aniel300** [reply]
so it just skip it?

**Reply by euzu** [reply]
yes skips until next scheduled update,
or manual update through web ui.

**Reply by aniel300** [reply]
ok perfect. that works for me
